### DISCUSSION TITLE
My MegaELS

### DISCUSSION THREAD

**Discussion by tonywright** [kachurovskiy/megaels#16] [Show and tell]
Pics of my MegaELS in place. The driver boards and psu are screwed to a board behind the lathe.
Found a grey petg plastic that is almost the exact colour of the paint used on the lathe so it matches in quite well. I recreated the front of the control box and printed in dual colour so the writing stands out. I also created and printed my own knobs for the rotary switches and VR.
Originally had an issue with the cones but reading the outstanding issue of HTML Tables lead me to the correct settings required.

![20221006_210218](https://user-images.githubusercontent.com/17741253/194408476-a32c6721-9ff7-4d4d-9d10-63ab10ece135.jpg)
![20221006_210200](https://user-images.githubusercontent.com/17741253/194408487-e2cbf57b-bf39-4659-9c59-631cda7d5260.jpg)

**Comment by kachurovskiy** [comment]
Awesome, congratulations on finishing the build! 🥳
